common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -59,29 +59,68 @@ switches.  Like with the earlier fences, another fence is erected 20m
 out which will again switch the vehicle to LAND (HOLD for Rover), if it continues away from
 home.
 
+Plane Actions
+-------------
+
+If, enabled, and  the fence radius (:ref:`FENCE_RADIUS<FENCE_RADIUS>`) or altitudes,if non-zero, (:ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` and :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>`) are exceeded, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` below:
+
++------------+--------------------+
++FENCE_ACTION|  Plane             +
++============+====================+
++     0      |   Report Only      +
++------------+--------------------+
++     1      | RTL                +
++------------+--------------------+
++     2      | Guided to return   +
++            | point              +
++------------+--------------------+
++     3      |  Guided to return  +
++            |  point with pilot  +
++            |  throttle control  +
++------------+--------------------+
+
+The guided return altitude during the return to the return point after breach, if this action is selected, is determined by :ref:`FENCE_RET_ALT<FENCE_RET_ALT>` . If either :ref:`FENCE_ACTION<FENCE_ACTION>` =2 or 3 is used. A fence return point **MUST** be setup or unpredictable action will occur (see video below). And the return point can be replaced by the nearest rally point as a destination, if :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` is enabled.
```

Comment:
Same as above, these `=2 or 3` is not accurate.

Reply:
With the PR that fixes the spiralling, is this comment still true?

`A fence return point **MUST** be setup or unpredictable action will occur (see video below)`

Reply:
fixed